docs/src/tutorials/linear/diet.jl

```diff
@@ -155,7 +146,8 @@ filter!(row -> row.quantity > 0.0, solution)
 # constraints. Let's see what happens if we add a constraint that we can buy at
 # most 6 units of milk or ice cream combined.
 
-@constraint(model, x["milk"] + x["ice cream"] <= 6)
+is_dairy = (foods.name .== "milk") .| (foods.name .== "ice cream")
```

Comment:
An equivalent alternative syntax that is more scalable for people working with sets is
```
dairy_foods = ["milk", "ice cream"]
is_dairy = foods.name .∈ [dairy_foods]
```